include/morphio/mut/morphology.h

```diff
@@ -210,8 +210,13 @@ class Morphology
        Fixes the morphology single child sections and issues warnings
        if the section starts and ends are inconsistent
      **/
-    void sanitize();
-    void sanitize(const morphio::readers::DebugInfo& debugInfo);
+    void removeUnifurcations();
+    void removeUnifurcations(const morphio::readers::DebugInfo& debugInfo);
+
```

Comment:
Maybe this would be helpful for some: 
```suggestion
bool hasUnifurcations()
```

Reply:
let's make another issue for that